lib/composer/package/loader/project_attributes_loader.rb

```diff
@@ -0,0 +1,56 @@
+module Composer

```

Comment:
I think it's dangerous to move into Composers namespace territory like this. Please house these under `Gitlab::Composer`.